msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -2292,4 +2271,35 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
         assertTrue(result is SignUpError)
         assertTrue((result as SignUpError).isInvalidUsername())
     }
+
+    @Test
+    fun testEmptyRequestParametersToGenericErrorNotThrownException() = runTest {
+        val correlationId = UUID.randomUUID().toString()
+
+        configureMockApi(
+            endpointType = MockApiEndpoint.SignUpStart,
+            correlationId = correlationId,
+            responseType = MockApiResponseType.SIGNUP_START_SUCCESS
+        )
+
+        configureMockApi(
+            endpointType = MockApiEndpoint.SignUpChallenge,
+            correlationId = correlationId,
+            responseType = MockApiResponseType.CHALLENGE_TYPE_OOB
+        )
+
+        val result = application.signUp(username)
+        assertTrue(result is SignUpResult.CodeRequired)
+
+        configureMockApi(
+            endpointType = MockApiEndpoint.SignUpContinue,
+            correlationId = correlationId,
+            responseType = MockApiResponseType.INVALID_OOB_VALUE
+        )
+
+        val submitCodeState = spy((result as SignUpResult.CodeRequired).nextState)
+        val submitCodeResult = submitCodeState.submitCode(emptyString)  // Empty code will trigger ArgUtils.validateNonNullArg(oob, "oob") of the SignUpContinueRequest to thrown ClientException
+        assertTrue(submitCodeResult is SubmitCodeError)
+        assertTrue((submitCodeResult as SubmitCodeError).error.equals(ErrorTypes.UNSUCCESSFUL_COMMAND_ERROR))  // ClientException will be caught in CommandResultUtil.kt and converted to generic error in interface layer
```

Comment:
Where is `ErrorTypes.UNSUCCESSFUL_COMMAND_ERROR` set?

Reply:
It set under  CommandResultUtil.kt `const val UNSUCCESSFUL_COMMAND_ERROR = "unsuccessful_command"`

Reply:
I don't think those changes are pushed, I don't see them in this PR.

Reply:
They originally existed in the common repository.
![image](https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/111055878/3052f55d-22d7-4671-8176-aa4e0f0bcf8a)

Reply:
It isn't used in the SDK code base but it is used in the appending test testEmptyRequestParametersToGenericErrorNotThrownException() and will be displayed in the sample app to user SubmitError(error= "unsuccessful_command")

Reply:
Offline discussion conclusion: Use direct string in the test file instead of adding new definition in the code base.